### ISSUE TITLE
"Permission denied" warnings are shown

### ISSUE THREAD

**Issue by ryonakano** [stsdc/monitor#154]
The following warinigs are shown in Terminal every one second since commit 1942b30a77989699cfa055d180ca8ddcf135fb2e:

```
** (com.github.stsdc.monitor:8299): WARNING **: 09:49:09.780: Process.vala:239: Error opening directory “/proc/1242/fd”: Permission denied

** (com.github.stsdc.monitor:8299): WARNING **: 09:49:09.780: Process.vala:239: Error opening directory “/proc/1262/fd”: Permission denied

** (com.github.stsdc.monitor:8299): WARNING **: 09:49:09.812: Process.vala:239: Error opening directory “/proc/1453/fd”: Permission denied

** (com.github.stsdc.monitor:8299): WARNING **: 09:49:09.837: Process.vala:239: Error opening directory “/proc/6413/fd”: Permission denied

** (com.github.stsdc.monitor:8299): WARNING **: 09:49:11.780: Process.vala:239: Error opening directory “/proc/1242/fd”: Permission denied
```

**Comment by stsdc** [comment]
yeah, I know... 😅️

**Comment by stsdc** [comment]
So it can't open files of these processes:
* sshd (basically ssh related processes)
* gnome-keyring
* (sd-pam)

Right now I don't how to fix this. Should probably display some info about error. In this case, probably, special signal should be added to Process.vala, that will emit info about error to UI layer. Then add special styled row to listbox with info of error.

**Comment by stsdc** [comment]
fixed in cb54ea10079351eb3b47d3a4a817f6eb8a0adf5d
